File: SDKs/optimade.local/1.1.0/typescript/src/sdk/models/shared/availableapiversion.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// AvailableApiVersion
/** 
 * A JSON object containing information about an available API version
**/
export class AvailableApiVersion extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=url" })
  url: string;

  @SpeakeasyMetadata({ data: "json, name=version" })
  version: string;
}

File: SDKs/optimade.local/1.1.0/python/src/sdk/models/shared/availableapiversion.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class AvailableAPIVersion:
    r"""AvailableAPIVersion
    A JSON object containing information about an available API version
    """
    
    url: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('url') }})
    version: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('version') }})
    
